com.unity.netcode.gameobjects/Tests/Runtime/AddNetworkPrefabTests.cs

```diff
@@ -22,17 +22,19 @@ protected override IEnumerator OnSetup()
             // Host is irrelevant, messages don't get sent to the host "client"
```

Comment:
@ShadauxCat what's your opinion on needing these change to this test ? I don't fully understand the test or why this is needed, but it seems to be necessary.